alpine/docker-entrypoint.sh

```diff
@@ -7,9 +7,17 @@ if [[ "$1" == "kong" ]]; then
   PREFIX=${KONG_PREFIX:=/usr/local/kong}
 
   if [[ "$2" == "docker-start" ]]; then
-    kong prepare -p "$PREFIX"
+    chown kong "$PREFIX"
+    su-exec kong kong prepare -p "$PREFIX"
+    
+    if [ ! -z ${KONG_STREAM_LISTEN+x} ]; then
+      setcap cap_net_raw=+ep /usr/local/openresty/nginx/sbin/nginx
+    fi
 
-    exec /usr/local/openresty/nginx/sbin/nginx \
+    chmod 777 /proc/self/fd/1
+    chmod 777 /proc/self/fd/2
+
+    su-exec kong /usr/local/openresty/nginx/sbin/nginx \
```

Comment:
This should be `exec su-exec`. otherwise the script will continue.

Reply:
That shouldn't be necessary (https://github.com/ncopa/su-exec/blob/master/su-exec.c#L103). It is also documented in the project's README: https://github.com/ncopa/su-exec#tty--parentchild-handling

Reply:
Unless you `exec` the bash process will live on as a parent.
The readme there is saying su-exec won't stick around; not the parent of su-exec.

Reply:
Indeed. Also, any thoughts on https://github.com/ncopa/su-exec/issues/23?